controllers/admin/AdminLoginController.php

```diff
@@ -201,6 +205,8 @@ public function processLogin()
 					$cookie->last_activity = time();
 
 				$cookie->write();
+				
+				Hook::exec('actionAdminAuthentication');
```

Comment:
actionAfterAdminAuthentication , it will be better, no ? ;-) (just to be sync with anothers hooks and before/after :))